### PR TITLE
Numbers and data types overview homework

### PR DESCRIPTION
**Pull request by Artemidium**

### CODE REVIEW

File: Exercises/numbers/numbers.js [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+const binary = 1110101010111;
+const octal = 16527;
+const hexadecimal = '1d57';
+
+console.log(parseInt(binary, 2));
+console.log(parseInt(octal, 8));
+console.log(parseInt(hexadecimal, 16));
+
+// Functions are not studied
 function extractNumbers (text) {
```
Comment (lizarusi) [inline Exercises/numbers/numbers.js:11]: we definitely not studied functions, but that's why you have a template 
```js
function extractNumbers(text) {
   const extractedNumer = <your code>;

   return extractedNumer;
}
```

**Review by lizarusi — COMMENTED** [review]
Everything is perfect, just add parsing numbers from string :)

**Review by Mike-Savin — APPROVED** [review]
👍

**Review by lizarusi — APPROVED** [review]
👍

**Pull request merged**